tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -7863,6 +7860,13 @@ bool SPIRVEmitter::processTessellationShaderAttributes(
                                 spv::ExecutionMode::OutputVertices,
                                 {*numOutputControlPoints});
   }
+  if (auto *pcf = decl->getAttr<HLSLPatchConstantFuncAttr>()) {
+    llvm::StringRef pcf_name = pcf->getFunctionName();
+    for (auto *decl : astContext.getTranslationUnitDecl()->decls())
+      if (auto *funcDecl = dyn_cast<FunctionDecl>(decl))
```

Comment:
Is it possible to have another non-pcf function with the same name? Should we also call `context.IsPatchConstantFunctionDecl(funcDecl)` here to make sure we are not using some non-pcf function?

Reply:
Good point... Done.